core/server/master/src/main/java/alluxio/master/scheduler/JournaledJobMetaStore.java

```diff
@@ -71,21 +70,9 @@ public boolean processJournalEntry(Journal.JournalEntry entry) {
       mExistingJobs.add(job);
     }
     if (entry.hasCopyJob()) {
-      Map<String, MountPointInfo> mountPointInfoSummary =
-          mFileSystemMaster.getMountPointInfoSummary(false);
-      UnderFileSystem ufsClient = null;
-      // no need to close under file system cause most close methods are noop
-      try {
-        ufsClient = mUfsManager
-            .get(mountPointInfoSummary.get(entry.getCopyJob().getSrcUfsAddress()).getMountId())
-            .acquireUfsResource().get();
-      } catch (NotFoundException | UnavailableException e) {
-        // should not happen in normal case, only happens when there's change in mount table during
-        // journal processing, but we should not fail the journal processing
-        LOG.warn("Failed to get ufs client for ufs address {}",
-            entry.getCopyJob().getSrcUfsAddress(), e);
-      }
-      Job<?> job = JobFactoryProducer.create(entry, ufsClient).create();
+      UnderFileSystem ufs = UnderFileSystem.Factory.create(entry.getCopyJob().getSrc(),
```

Comment:
For each copy job we create a UFS? do we close the UFS? are there potentially resource leak?

Reply:
I think we need ufsmanager without mountId, would do that in another PR